src/components/atoms/Counter.svelte

```diff
@@ -0,0 +1,17 @@
+<script>
+</script>
+
+<style lang="scss">
+  // Import fonts, vars, etc.
+  @import 'src/styles/index.scss';
+
+  h1 {
+    font-size: 2em;
+    color: $ui-black;
+  }
+
+</style>
+
+<h1>
+  <slot />
+</h1>
```

Comment:
I wouldn't make this as an h1 (since there is already an h1 on the page). You'd better make a p tag with a class and make it bigger with styling (like font-size and weight).